docs/src/main/sphinx/functions/ai.md

```diff
@@ -0,0 +1,136 @@
+# AI functions
+
+## Overview
+
+The AI functions allow you to invoke a large language model (LLM)
+to perform various textual tasks.
+Multiple LLM providers are supported: OpenAI and Ollama.
+
+## Configuration
```

Comment:
I had a look at the code now .. there are a bunch of config properties.. we should add a table with all of them ..

Reply:
The current content is sufficient to get started. We can do a follow up PR for more docs and examples